tests/integration/security/util/reachability/context.go

```diff
@@ -86,14 +91,26 @@ func CreateContext(ctx framework.TestContext, p pilot.Instance) Context {
 			Annotations: echo.NewAnnotations().SetBool(echo.SidecarInject, false),
 		},
 	}
-	echoboot.NewBuilderOrFail(ctx, ctx).
+
+	// VM specific setup
+	vmCfg := util.EchoConfig("vm", ns, false, nil, p)
+	vmCfg.DeployAsVM = true
```

Comment:
Should vmCfg.DeployAsVM be configured based on the buildVM input parameter?

Reply:
Thanks for your comments! That's a great idea! Will update!